README.md

```diff
@@ -17,25 +16,43 @@ Join [#events-app-beta](https://hubspotdev.slack.com/archives/C011GFF8KNZ) in th
 
 0. Make sure that you're set up for [local development](https://designers.hubspot.com/tutorials/getting-started) with the [HubSpot CMS CLI](https://designers.hubspot.com/docs/developer-reference/local-development-cms-cli).
 1. Clone this repo to your machine
-2. Install dependencies by running `yarn install`
-3. Run `yarn create-table` to create the HubDB table where you will manage your events
-4. Add your [HubSpot API key](https://knowledge.hubspot.com/integrations/how-do-i-get-my-hubspot-api-key) for the portal by running `yarn hs secrets add APIKEY <api-key-goes-here>`. The API key is used by `eventSignup.js` to update the HubDB table
-5. Run `yarn start` which will build the javascript, auto-upload the files to your `defaultPortal`, and watch for changes
-6. Create a page in your portal that:
-    - Has the slug `/events/`
-    - Uses the `Events app` module in a page template
-    - References the `Events` HubDB table, set up for dynamic pages
-
-7. Registrations are stored in the CRM using the [Forms API](https://developers.hubspot.com/docs/methods/forms/forms_overview). For each event that you create, you'll need to create a form and add that form ID to the HubDB entry. The form fields needed are:
-    - First name
-    - Last name
-    - Email
+1. Install dependencies by running `yarn install`
+1. Run `yarn create-table` to create the HubDB table where you will manage your events
+1. Add your [HubSpot API key](https://knowledge.hubspot.com/integrations/how-do-i-get-my-hubspot-api-key) for the portal by running `yarn hs secrets add APIKEY <api-key-goes-here>`. The API key is used by `eventSignup.js` to update the HubDB table
+1. Run `yarn start` which will build the javascript, auto-upload the files to your `defaultPortal`, and watch for changes
+1. Create a page in your portal that:
+
+   - Has the slug `/events/`
+   - Uses the `Events app` module in a page template
+   - References the `Events` HubDB table, set up for dynamic pages
+
+1. Registrations are stored in the CRM using the [Forms API](https://developers.hubspot.com/docs/methods/forms/forms_overview). For each event that you create, you'll need to create a form and add that form ID to the HubDB entry. The form fields needed are:
```

Comment:
Should these all be `1`?

Reply:
Thanks for pointing this out! I'll address this--prettier keeps changing these to all 1.

Reply:
fwiw, I'm pretty sure the actual number in markdown ordered lists doesn't matter, so:
```
1. Foo
1. Bar
1. Foobar
```
is expected to render as:
1. Foo
1. Bar
1. Foobar

Reply:
Confirmation: https://www.markdownguide.org/basic-syntax/#ordered-lists

Reply:
Thanks for pointing this out, @levinkeo! In this instance, I wanted to keep our original numbering, because our list begins with Step 0 instead of 1. I had some issues early on in these edits, when Step 0 was being automatically converted to Step 1.